pywt/multilevel.py

```diff
@@ -182,7 +182,7 @@ def waverec2(coeffs, wavelet, mode='sym'):
     """
     Multilevel 2D Inverse Discrete Wavelet Transform.
 
-    coeffs : array_like
+    coeffs : list
```

Comment:
It accepts tuple as well, so this should either stay as is or change to ``list or tuple``.